Packs/SumoLogic_Cloud_SIEM/Classifiers/classifier-Sumo_Logic_Cloud_SIEM.json

```
@@ -1,34 +1,17 @@
 {
 	"defaultIncidentType": "",
 	"definitionId": "",
-	"description": "Classifier for Sumo Logic Insight",
+	"description": "Classifier for Sumo Logic Insight and Signal",
 	"feed": false,
 	"fromVersion": "6.2.0",
-	"id": "Sumo Logic Insight Classifier",
+	"id": "Sumo Logic Cloud SIEM Classifier",
 	"keyTypeMap": {
-		"INSIGHT": "Sumo Logic Insight"
+		"INSIGHT": "Sumo Logic Insight",
+		"SIGNAL-": "Sumo Logic Signal"
 	},
-	"name": "Sumo Logic Insight Classifier",
+	"name": "Sumo Logic Cloud SIEM Classifier",
```

As discussed on the demo, please revert the name/ID from the classifier.

```suggestion
	"id": "Sumo Logic Insight Classifier",
```

---

Does the file name of the classifier matter? I added a new file called "classifier-Sumo_Logic_Insight.json" with the nam+id= "Sumo Logic Insight Classifier" (and the same content). As discussed, I want to use the name "Sumo Logic Cloud SIEM Classifier" to be a new classifier, however if I name the file "classifier-Sumo_Logic_Cloud_SIEMv2.json", demise-sdk will complain the classifier v2 doesn't exist so I added a new file above and keep this classifier name+id="Sumo Logic Cloud SIEM Classifier".

---

That new file is this: https://github.com/demisto/content/pull/25414/files#diff-098e0fec9d7455f0a5a0b7cd8dcfc09b01c9648e8d1049bc4a05695a13ea9ff4, and other than the filename, its content is almost identical to the old "Sumo Logic Insight Classifier" https://github.com/demisto/content/blob/master/Packs/SumoLogic_Cloud_SIEM/Classifiers/classifier-Sumo_Logic_Cloud_SIEM.json

---

@kgal-pan: could you confirm if this approach is ok ^^^, or do I have to keep the same file name and revert to the old ID+Name? In that case, how do I add the new classifier with the name+id= "Sumo Logic Cloud SIEM Classifier"? 

Have merged with latest master branch as requested

---

Hey @duchatran, your PR review is slightly delayed due to holidays in Israel - I apologize.
You were told to revert the change of the ID and name because:
1. IDs and names must equal
2. we don't allow changing IDs of existing content items (it can cause issues during the pack update in XSOAR).

However, the names of the files do not really matter - I suggest renaming this one to `classifier-Sumo_Logic_Insight.json`, and the new one to `classifier-Sumo_Logic_Cloud_SIEM.json`. I hope this helps.

---

Thanks, I'll go with this suggestion.

---

Name and id have been reverted.